app/services/users.js

```diff
@@ -40,8 +44,19 @@ exports.findAllUsers = (limit, offset) =>
   User.findAndCountAll({
     limit,
     offset,
-    attributes: ['id', 'firstName', 'lastName', 'email']
+    attributes: ['id', 'firstName', 'lastName', 'email', 'role']
   }).catch(() => {
     logger.info('Error trying to find the users');
     throw errors.databaseError('Error processing request in database.');
   });
+
+exports.changeRole = email =>
+  User.update(
+    { role: ADMIN_ROLE },
+    {
+      where: { email, role: REGULAR_ROLE }
```

Comment:
The condition role is unnecessary, email  is  unique

Reply:
I do this so that it does not always update